CHANGELOG.rst

```diff
@@ -5,6 +5,39 @@ The cloud.terraform collection Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
+Major Changes
+-------------
+
+- Added search_child_modules option to terraform_provider inventory plugin.
```

Comment:
```suggestion
- terraform_provider - Added ``search_child_modules`` option.
```

Reply:
Can you also please link the PR where this feature has been added?

Reply:
I will do it, but we are supposed to track this during PR review